src/aids/resources.py

```diff
@@ -102,7 +102,7 @@ def before_import_row(self, row, **kwargs):
             if key in row:
                 del row[key]
         # add/set keys
-        if 'projects' not in row:
+        if not 'projects' in row:
```

Comment:
```suggestion
        if 'projects' not in row:
```
L'original marchait, là ca fait planter